### PR TITLE
Edit Solution Features

### PR DESCRIPTION
**Pull request by kiran-mastek**
Story: AB#15048
Task: AB#15757

### CODE REVIEW

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Admin/Views/CatalogueSolutions/Features.cshtml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                    <span role="text">
+                        Features
+                        <span class="nhsuk-caption-xl nhsuk-caption--bottom">
+                            <span class="nhsuk-u-visually-hidden"> - </span>
+                            @Model.SolutionName
+                        </span>
+                    </span>
+                </h1>
+
+                <p class="nhsuk-lede-text">Highlight to buyers the key features of your Catalogue Solution.</p>
```
Comment (RahulJha-mastek) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Admin/Views/CatalogueSolutions/Features.cshtml:27]: We can use new UI template for this title.
Suggestion:
```
<nhs-page-title title="Features"
                            caption="@Model.SolutionName"
                            advice="Highlight to buyers the key features of your Catalogue Solution." />
```

File: tests/NHSD.GPIT.BuyingCatalogue.WebApp.UnitTests/Areas/Admin/Controllers/CatalogueSolutionsControllerTests.cs [hunk 2]
```diff
@@ -41,6 +44,103 @@ public static void Constructor_NullService_ThrowsException()
...
+        [Theory]
+        [CommonAutoData]
+        public static async Task Get_Features_ValidId_ReturnsViewWithExpectedModel(
+            CatalogueItem catalogueItem,
+            CatalogueItemId catalogueItemId,
+            [Frozen]Mock<ISolutionsService> mockService)
+        {
+            mockService.Setup(s => s.GetSolution(catalogueItemId))
+                .ReturnsAsync(catalogueItem);
+            var controller = new CatalogueSolutionsController(mockService.Object, Mock.Of<IUsersService>());
```
Comment (RahulJha-mastek) [inline tests/NHSD.GPIT.BuyingCatalogue.WebApp.UnitTests/Areas/Admin/Controllers/CatalogueSolutionsControllerTests.cs:72]: Can we remove `controller` variable if not used.

File: tests/NHSD.GPIT.BuyingCatalogue.WebApp.UnitTests/Areas/Admin/Controllers/CatalogueSolutionsControllerTests.cs [hunk 3]
```diff
@@ -41,6 +44,103 @@ public static void Constructor_NullService_ThrowsException()
...
 
+        [Theory]
+        [CommonAutoData]
+        public static async Task Get_Features_GetsSolutionFromService(
+            CatalogueItemId catalogueItemId,
+            [Frozen]Mock<ISolutionsService> mockService)
+        {
+            mockService.Setup(s => s.GetSolution(catalogueItemId))
+                .ReturnsAsync(new CatalogueItem());
+            var controller = new CatalogueSolutionsController(mockService.Object, Mock.Of<IUsersService>());
```
Comment (RahulJha-mastek) [inline tests/NHSD.GPIT.BuyingCatalogue.WebApp.UnitTests/Areas/Admin/Controllers/CatalogueSolutionsControllerTests.cs:55]: Can we remove `controller `variable if not used.

File: tests/NHSD.GPIT.BuyingCatalogue.WebApp.UnitTests/Areas/Admin/Controllers/CatalogueSolutionsControllerTests.cs [hunk 4]
```diff
@@ -1,13 +1,16 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Reflection;
```
[the remark this answers is not in the document]
Comment (kiran-mastek) [reply tests/NHSD.GPIT.BuyingCatalogue.WebApp.UnitTests/Areas/Admin/Controllers/CatalogueSolutionsControllerTests.cs:4]: removed

File: tests/NHSD.GPIT.BuyingCatalogue.WebApp.UnitTests/Areas/Admin/Controllers/CatalogueSolutionsControllerTests.cs (hunk 3 above)
Comment (kiran-mastek) [reply tests/NHSD.GPIT.BuyingCatalogue.WebApp.UnitTests/Areas/Admin/Controllers/CatalogueSolutionsControllerTests.cs:55]: updated

**Review by RahulJha-mastek — APPROVED** [review]

**Pull request merged**